edivorce/apps/core/templates/dashboard/help_scanning.html

```diff
@@ -0,0 +1,130 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ block.super }}: Overview{% endblock %}
+
+{% block progress %}
+    {% include "partials/dashnav.html" with active_page=active_page %}
+{% endblock %}
+
+{% block content %}
+    <h1>Get Help Opening and Saving PDF forms</h1>
+    <div>
+        <h3>Your divorce forms</h3>
+
+        <p>You can review, print, or save your forms by clicking on the
+            &ldquo;Review and Print&rdquo; button next to each form. This will open or
+            download a PDF version of the form.</p>
+
+        <p>&ldquo;PDF&rdquo; stand for Portable Document Format, which is a file
```

Comment:
stand**s** for (missing s)

Reply:
Oh yeah, this is a dummy page and I will fill it in later. But good catch. The content is from another page in the app ("Help saving PDF")